src/scripts/utils.py

```diff
@@ -280,7 +279,7 @@ def _get_all_owned_terms(
         direct_parents_df: pd.DataFrame = jinja_sparql(
             onto_path=ontology_path,
             jinja_path=os.path.join(PROJECT_DIR, 'src', 'sparql', 'get-terms-direct-parents.sparql.jinja2'),
-            prefix_map=owned_prefix_map, use_cache=True, values=' '.join(query_terms))
+            prefix_map=owned_prefix_map, use_cache=use_cache, values=' '.join(query_terms))
```

Comment:
@matentzn This was the main issue. I feel bad about this somehow slipping in. It was previously always using the cache. The cache has still saved me a lot of time in the past, but this one did take a few hours for me to fix as well as you all's attention.